### PR TITLE
LPS-195634 Migrate Page Editor Filter by Content Type Select to Clay Picker

### PR DESCRIPTION
**Pull request by claraizquierdo**
Motivation
=======
[Link to story or ticket](https://liferay.atlassian.net/browse/LPS-195634)

Steps to Verify:
----------------
1. Go to the page editor
2. On the left sidebar click on Page Content icon (the clip)

Screenshots (optional):
-----------------------

Captura de pantalla 2023-09-08 a las 12 17 35

### CODE REVIEW

**Comment by liferay-echo** [comment]
Thank you for submitting this PR. 👏

Make sure it follows the [Echo Pull Request Template](https://gist.github.com/liferay-echo/5a4455285ff1800f9da146816d85e67d#file-pull-request-template-md) if you haven't already done so. 

Learn more about it [here](https://liferay.atlassian.net/wiki/spaces/ENGECHO/pages/2027455014/Echo+Pull+Request+Etiquette).

**Comment by claraizquierdo** [comment]
ci:test:sf

File: modules/apps/layout/layout-content-page-editor-web/src/main/resources/META-INF/resources/page_editor/plugins/page_content/components/ContentFilter.js [hunk 1]
```diff
@@ -14,50 +14,37 @@ export default function ContentFilter({
...
-			<ClayDropDown
-				alignmentPosition={Align.BottomLeft}
-				className="mb-2"
-				closeOnClick
-				menuElementAttrs={{
-					containerProps: {
-						className: 'cadmin',
-					},
+			<Picker
+				UNSAFE_menuClassName="cadmin"
```
Comment (victorg1991) [inline modules/apps/layout/layout-content-page-editor-web/src/main/resources/META-INF/resources/page_editor/plugins/page_content/components/ContentFilter.js:38]: @DiegoHu97 I think this should be added in the one about edition mode, could you take a look, please??

**Comment by victorg1991** [comment]
Just minor stuff @claraizquierdo :D

File: modules/apps/layout/layout-content-page-editor-web/src/main/resources/META-INF/resources/page_editor/plugins/page_content/components/ContentFilter.js [hunk 2]
```diff
@@ -14,50 +14,37 @@ export default function ContentFilter({
...
+	const Trigger = React.forwardRef((otherProps, ref) => (
+		<ClayButton
+			{...otherProps}
+			aria-label={Liferay.Language.get('filter-by-content-type')}
+			className="form-control form-control-select form-control-sm mb-2 text-left"
+			displayType="unstyled"
+			ref={ref}
+			size="sm"
+			type="button"
+		>
```
[the remark this answers is not in the document]
Comment (claraizquierdo) [reply modules/apps/layout/layout-content-page-editor-web/src/main/resources/META-INF/resources/page_editor/plugins/page_content/components/ContentFilter.js:26]: Hi @victorg1991 , the `displayType` and `className` properties are necessary, but not the other ones so I'm going ti remove them (`size` and `type`).

File: modules/apps/layout/layout-content-page-editor-web/src/main/resources/META-INF/resources/page_editor/plugins/page_content/components/ContentFilter.js [hunk 3]
```diff
@@ -14,50 +14,37 @@ export default function ContentFilter({
...
-				closeOnClick
-				menuElementAttrs={{
-					containerProps: {
-						className: 'cadmin',
-					},
+			<Picker
+				UNSAFE_menuClassName="cadmin"
+				as={Trigger}
+				items={contentTypes}
+				onSelectionChange={(type) => {
```
[the remark this answers is not in the document]
Comment (claraizquierdo) [reply modules/apps/layout/layout-content-page-editor-web/src/main/resources/META-INF/resources/page_editor/plugins/page_content/components/ContentFilter.js:41]: Sure!

**Comment by claraizquierdo** [comment]
> Just minor stuff @claraizquierdo :D

I just applied the changes :)

**Comment by claraizquierdo** [comment]
ci:forward

**Comment by Tim-Cao** [comment]
See https://github.com/liferay-echo/liferay-portal/pull/13657

**Comment by Tim-Cao** [comment]
ci:close

**Pull request closed without merging**